cmd/wego/flux/cmd.go

```diff
@@ -0,0 +1,34 @@
+package flux
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+//go:embed flux
+var flux []byte
+
+var Cmd = &cobra.Command{
+	Use:   "flux",
+	Short: "Use flux commands",
+	Run:   runCmd,
+}
+
+func init() {
+	_ = os.WriteFile("bin/flux", flux, 0755)
```

Comment:
Not sure where this is supposed to write the binary... We don't have a bin directory at install time (that I"m aware of). Maybe this should go in a temp directory?

Reply:
can't we run it from memory?